front/components/actions/mcp/RemoteMCPForm.tsx

```diff
@@ -178,19 +186,58 @@ export function RemoteMCPForm({ owner, mcpServer }: RemoteMCPFormProps) {
       <Separator className="my-4" />
 
       <Page.SectionHeader title="Settings" />
-      <div>
-        <Label htmlFor="name">Name</Label>
-        <Controller
-          control={form.control}
-          name="name"
-          render={({ field }) => (
-            <Input
-              {...field}
-              isError={!!form.formState.errors.name}
-              message={form.formState.errors.name?.message}
-            />
-          )}
-        />
+      <div className="flex space-x-2">
+        <div className="flex-grow">
+          <Label htmlFor="name">Name</Label>
+          <Controller
+            control={form.control}
+            name="name"
+            render={({ field }) => (
+              <Input
+                {...field}
+                isError={!!form.formState.errors.name}
+                message={form.formState.errors.name?.message}
+              />
+            )}
+          />
+        </div>
+
+        <div>
+          <Label htmlFor="icon">Icon</Label>
```

Comment:
we'll need a full icon list and a nice icon picker (although this can be done in another pr)